elide-model-config/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/jsonformats/ElideClassNameFormatAttr.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.jsonformats;
+
+import com.github.fge.jackson.NodeType;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.core.report.ProcessingReport;
+import com.github.fge.jsonschema.format.AbstractFormatAttribute;
+import com.github.fge.jsonschema.processors.data.FullData;
+import com.github.fge.msgsimple.bundle.MessageBundle;
+
+public class ElideClassNameFormatAttr extends AbstractFormatAttribute {
+    private static final String CLASS_NAME_FORMAT_REGEX = "^([a-zA-Z]+[a-zA-Z0-9_]*\\.)+class$";
+
+    public static final String FORMAT_NAME = "elideClassName";
+    public static final String FORMAT_KEY = "elideClassName.error.format";
+    public static final String FORMAT_MSG = "Class Name [%s] is not allowed. "
+                    + "Class name must follow the format 'X.X.X.X.class'. "
```

Comment:
Something simple here might be better:  "[%s] is not a valid Java class name."

Reply:
updated.